modules/gitops-argo-cd-properties.adoc

```diff
@@ -10,40 +10,101 @@
+  * _<Policy>_ - The `policy.csv` property in the `argocd-rbac-cm` ConfigMap. CSV data containing user-defined RBAC policies and role definitions.
+  * _<Scopes>_ - The scopes property in the `argocd-rbac-cm` ConfigMap. Controls which OIDC scopes to examine during RBAC enforcement (in addition to sub scope).
+|`Redis` |Redis configuration options.|__<Object>__
+a|* _<AutoTLS>_ - Use the provider to create the redis server's TLS certificate (one of: openshift). Currently only available for {product-title}.
+  * _<DisableTLSVerification>_ - Define whether the redis server should be accessed using strict TLS validation.
+  * _<Image>_ - The container image for Redis. This overrides the ARGOCD_REDIS_IMAGE environment variable.
+  * _<Resources>_ - The container compute resources.
+  * _<Version>_ - The tag to use with the Redis container image.
+|`ResourceCustomizations` |Customize resource behavior.|__<empty>__|
+|`ResourceExclusions` |Completely ignore entire classes of resource group.|__<empty>__|
+|`ResourceInclusions` |The configuration to configure which resource group/kinds are applied.|__<empty>__|
+|`Server` |Argo CD Server configuration options.|__<Object>__
+a|* _<Autoscale>_ - Server autoscale configuration options.
+  * _<ExtraCommandArgs>_ - List of arguments added to the existing arguments set by the Operator.
+  * _<GRPC>_ - GRPC configuration options.
+  * _<Host>_ - The hostname used for Ingress/Route resources.
+  * _<Ingress>_ - Ingress configuration for the Argo CD server component.
+  * _<Insecure>_ - Toggles the insecure flag for Argo CD server.
+  * _<Resources>_ - The container compute resources.
+  * _<Replicas>_ - The number of replicas for the Argo CD server. Must be greater than equal to 0. If Autoscale is enabled, Replicas is ignored.
+  * _<Route>_ - Route configuration options.
+  * _<Service.Type>_ - The `ServiceType` used for the service resource.
+  * _<LogLevel>_ - The log level to be used by the ArgoCD Server component. Valid options are debug, info, error, and warn.
+  * _<LogFormat>_ - The log format used by the ArgoCD Application Controller component. Valid options are text or json.
+  * _<Env>_ - Environment to set for the server workloads.
+|`SSO` |Single sign-on options.|__<Object>__
+a|* _<Image>_ - The container image for keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
+  * _<Keycloak>_ - Configuration options for Keycloak SSO provider.
+  * _<Dex>_ - Configuration options for Dex SSO provider.
+  * _<Provider>_ - The name of the provider used to configure Single sign-on. For now the supported options are Dex and keycloak.
+  * _<Resources>_ - The container compute resources.
+  * _<VerifyTLS>_ - Whether to enforce strict TLS checking when communicating with Keycloak service.
+  * _<Version>_ - The tag to use with the keycloak container image.
```

Comment:
"Keycloak" to be consistent

Reply:
I still see lowercase keycloak, but think most of them should be uppercase if it is the product that is being referred to